### ISSUE TITLE
Extraneous characters passed in argument to 

### ISSUE THREAD

**Issue by 22459** [longld/peda#118]
Running my executable with
```
start `echo -ne 'AAAAAAAAAAABBBBBBBBBB\xf7\xd7\xff\xffCCCCCC'`
```
results in extraneous characters, in this case the `\303` (`0xc3`) in the argument string:
 `EAX: 0xffffd7f6 ('A' <repeats 11 times>, "BBBBBBBBBB\303\267\303\227\303\277\303\277CCCCCC")`
This problem isn't present in bare bones gdb so I think it's a PEDA issue.

**Comment by longld** [comment]
Yes, it's a bug with `start` command. Workaround: use `run` instead :)
